it-index.html

```diff
@@ -0,0 +1,63 @@
+---
+title: "Jamulus ‒ Suonare Online. Tra Amici. In libertà."
+lang: "it"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Music"
```

Comment:
Looks good. 

Is the Italian word for music the same as in English?

Reply:
I applied the correction on the index page. Thanks!